helm-charts/common/ui/templates/configmap.yaml

```diff
@@ -43,4 +43,6 @@ data:
   BASE_URL: {{ .Values.BACKEND_SERVICE_ENDPOINT | quote }}
   {{- else if contains "agent-ui" .Values.image.repository }}
   AGENT_URL: {{ .Values.BACKEND_SERVICE_ENDPOINT | quote }}
+  {{- else if contains "productivity-suite-ui" .Values.image.repository }}
+  APP_KEYCLOAK_SERVICE_ENDPOINT: {{ .Values.APP_KEYCLOAK_SERVICE_ENDPOINT | quote }}
```

Comment:
APP_KEYCLOAK_SERVICE_ENDPOINT's default value should be added to the values.yaml in ui chart

Reply:
ok. I saw all of them are added to `variant_producitity_suite-values.yaml`